components/engine_tirocks/src/properties/mvcc.rs

```diff
@@ -151,14 +151,14 @@ impl TablePropertiesCollector for MvccPropertiesCollector {
 
         // Add new row.
         if self.row_versions == 1 {
-            self.cur_index_handle.size += 1;
-            self.cur_index_handle.offset += 1;
-            if self.cur_index_handle.offset == 1
-                || self.cur_index_handle.size >= PROP_ROWS_INDEX_DISTANCE
+            self.cur_prop_index.prop += 1;
+            self.cur_prop_index.offset += 1;
```

Comment:
Does this mean the `prop` covers a range ends with `offset` (instead of starting from `offset`)

Reply:
Yes.